packages/tangle/timemanager.go

```diff
@@ -181,6 +217,7 @@ func (t *TimeManager) mainLoop() {
 type LastConfirmedMessage struct {
 	MessageID MessageID `serix:"0"`
 	Time      time.Time `serix:"1"`
+	ConfirmedTime time.Time `serix:"2"`
```

Comment:
property naming is a little confusing - which one stores time of a message and which one the time when the message is confirmed? Would be nice to at least have some comment here.

Also update the comment on the whole struct.

Reply:
Done